src/disco/gui/fd_gui_printf.h

```diff
@@ -0,0 +1,91 @@
+#include "fd_gui.h"
+
```

Comment:
Could you include a doc comment about these? You don't need an individual comment for each function. E.g. all these that take no parameters could be grouped